### PR TITLE
LPS-179356 [PRM] Welcome banner does not remain hidden on refresh

### PR DESCRIPTION
**Pull request by kiwm**
Related Issue: https://issues.liferay.com/browse/LPS-176930

### CODE REVIEW

**Comment by kiwm** [comment]
ci:test:relevant

**Review by kevenleone — COMMENTED** [review]
LGTM @kiwm 

I just don't saw any relationship between fragment and `liferay.js` file

is it being used?

**Comment by kiwm** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/content-welcome-banner/index.html [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-<div class="align-items-center bg-neutral-1 d-lg-flex p-4 pre-dismiss rounded-xl">
+<div class="align-items-center bg-neutral-1 d-none p-4 pre-dismiss rounded-xl">
 	<div class="col">
-		<h1 data-lfr-editable-id="first-welcome-banner-title" data-lfr-editable-type="rich-text">
+	<h1 data-lfr-editable-id="first-welcome-banner-title" data-lfr-editable-type="rich-text">
 			This is editable text!
 		</h1>
 
-		<p class="align-items-center d-flex" data-lfr-editable-id="first-welcome-banner-description" data-lfr-editable-type="rich-text">
+	<p class="align-items-center d-flex" data-lfr-editable-id="first-welcome-banner-description" data-lfr-editable-type="rich-text">
```
Comment (claricebrasil) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/content-welcome-banner/index.html:13]: identation issue

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/content-welcome-banner/index.html [hunk 2]
```diff
@@ -26,15 +26,20 @@ <h1 data-lfr-editable-id="first-welcome-banner-title" data-lfr-editable-type="ri
 	</div>
 </div>
 
-<div class="align-items-center d-none justify-content-between p-4 pos-dismiss">
+<div class="align-items-center d-lg-flex justify-content-between p-4 pos-dismiss">
 	<div class="col">
-		<h1 data-lfr-editable-id="second-welcome-banner-title" data-lfr-editable-type="rich-text">
+	<h1 data-lfr-editable-id="second-welcome-banner-title" data-lfr-editable-type="rich-text">
```
Comment (claricebrasil) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/content-welcome-banner/index.html:31]: identation issue

**Comment by claricebrasil** [comment]
ci:forward

**Comment by claricebrasil** [comment]
ci:forward

**Pull request closed without merging**